components/sections/homepage/About.jsx

```diff
@@ -13,49 +13,49 @@ export default function About() {
   // this function used to set undeline line on all tabs
   function undelineStyle() {
     let defaultStyle =
-      "transition-all ease-in-out duration-300 pt-2 absolute border-b-1.5  bottom-0 border-purple ";
+      "transition-all ease-in-out duration-300 mt-2 absolute h-[1.5px] bg-purple  bottom-0  ";
```

Comment:
Avoid using  arbitrary value ,better this 1.5px should be configured in tailwind config